packages/connect-popup/webpack/prod.webpack.config.ts

```diff
@@ -1,7 +1,7 @@
 import path from 'path';
 import webpack, { DefinePlugin } from 'webpack';
 import HtmlWebpackPlugin from 'html-webpack-plugin';
-import TerserPlugin from 'terser-webpack-plugin';
+// import TerserPlugin from 'terser-webpack-plugin';
```

Comment:
this is temporary as well. I needed to check of code looks unminimized. removing it now